master/pkg/tasks/task.go

```diff
@@ -79,6 +79,7 @@ type TaskSpec struct {
 	Devices                []device.Device
 
 	UserSessionToken string
+	TaskType         model.TaskType
```

Comment:
if we won't check for tasktype on agent, this probably won't be necessary as well